doc/content/verification_and_validation/ver-1kb.md

```diff
@@ -0,0 +1,83 @@
+# ver-1kb
+
+# Henry’s Law Boundaries with No Volumetric Source
+
+## General Case Description
+
+Two enclosures are separated by a membrane that allows diffusion according to Henry’s law, with no volumetric source present. Enclosure 2 has twice the volume of Enclosure 1.
+
+## Case Set Up
+
+This verification problem is taken from [!cite](ambrosek2008verification).
+
+Over time, the pressures of T$_2$, which diffuses across the membrane in accordance with Henry’s law, will gradually equilibrate between the two enclosures.
```

Comment:
You sometimes call the gas H_2 (test specification file, for example), sometimes T_2 (here and in the figures, for example). It's applicable to both, but please be consistent. I have a very slight preference for T_2 here.

Reply:
@Lee01Atom please do not manually mark conversations as resolved, as I prefer to do it myself to keep track of what has been addressed and what hasn't. 
It is fine to have comments being marked as resolved automatically when you apply a suggestions, but please do not resolve them by hand. 
Thank you!